src/modal/SaveModal.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { useContext, useEffect } from 'react';
```

Comment:
The SaveModal has been replaced in NeoDash 2.4.
We should inject the `SaveToHiveModal` in the new sidebar and delete this file

Reply:
Agreed.  I can remove it and add the `SaveToHiveModal` injection

Reply:
Deleted SaveModal.tsx
The DashboardSidebar and DashboardSidebarExportModal already have references to connectionModule.getPublishUIDialog